auth-web/src/components/auth/staff/PPRLauncher.vue

```diff
@@ -39,8 +38,7 @@ export default class PPRLauncher extends Vue {
   }
 
   private getImgUrl (img) {
-    const images = require.context('@/assets/img/')
```

Comment:
Travis, there's a package that fixes this. I probably should've mentioned that to you, my bad. 
https://github.com/bcgov/business-create-ui/blob/3c85ddb387a7dc7b3f94e14194886778ad8f0b15/vite.config.js#L34

Reply:
I tried it, didn't work for me

Reply:
Oh OK, I see.